api/openapi3_0.yaml

```diff
@@ -0,0 +1,950 @@
+openapi: 3.0.0
+info:
+  title: Pandemos
+  description: >-
+    API for visualization of Infection Models
+  version: "1"
+security:
+  - bearerAuth: []
+servers:
+  - url: 'https://api.example.com/api/v0'
+  - url: 'http://localhost:8080/'
+paths:
+  /scenarios/:

```

Comment:
Can we remove all the trailing slashes from the endpoints?
From the OpenAPI Spec they are not necessary and it would allow us to use [https://api-platform.com/docs/admin/](https://api-platform.com/docs/admin/) to test and fill the API.